### ISSUE TITLE
TX Security Providers: Warning Message

### ISSUE THREAD

**Issue by ElvirCe** [MetaMask/metamask-extension#16258]

**Comment by ElvirCe** [comment]
@bschorchit the original requirement was to create only one message with the copy: “OpenSea has reviewed this request and is flagging it as malicious. Approving it could lead to a loss of funds. Proceed with caution.”
I see that the design has been updated and will put the ticket on hold until we groom it out and clarify everything.

**Comment by ElvirCe** [comment]
@bschorchit we will need the URL for the OpenSea link and also please note that the messages are a new component since we don't have them anywhere like this so the scope of the ticket will have to be expanded.

**Comment by bschorchit** [comment]
url is this: https://opensea.io/

**Comment by bschorchit** [comment]
> please note that the messages are a new component since we don't have them anywhere like this so the scope of the ticket will have to be expanded.
Thanks for pointing this out, Elvir. I agree with expanded scope, but just want to have this component reviewed by design system team before we add it. 
@georgewrmarshall could you kindly confirm if the warning component being used in the design matches our patterns and design system before we go ahead with it?
Here's the link for the designs: https://www.figma.com/file/Q69vHaA7Or7Rxz4vdzjVHC/Support-for-tx-security-providers-(MVP)?node-id=4986%3A26901&t=H3PgEpqvjHPr1hnS-0

**Comment by ElvirCe** [comment]
We are blocked on this ticket until the Banner component gets merged.

**Comment by bschorchit** [comment]
Banner has been merged, @ElvirCe
Thanks, @garrettbear!
